examples/permissioned-token/README.md

```diff
@@ -96,3 +96,50 @@ Developer hour links:
 - *implementation of the solution above using algob*: [examples/restricted-assets](/examples/restricted-assets)
 
 Link to spec of current template: [https://paper.dropbox.com/doc/Algob-Security-Token-Template-FR2LXhVg3edevYPBQZw6F](https://paper.dropbox.com/doc/Algob-Security-Token-Template-FR2LXhVg3edevYPBQZw6F)
+
+### API Doc
+Below we describe different use cases. We describe a use case using a function notation (along with transaction group structure). We implement that function in scripts, but for direct integration in smart contracts, you need to construct transactions as described below.
+
+#### Common
+
+1. [*totalSupply(assetIndex)*](examples/permissioned-token/scripts/common/common.js): returns total supply of the asset (*asset.total* - *asaReserveHolding.amount*).
+
+#### Admin
+1. [*issue(address, amount)*](examples/permissioned-token/scripts/admin/issue.js): Group of 2 transactions
+   - *tx1*: Call to controller smart contract with a) application arg `str:issue` b) foreign Asset: `assetIndex` - signed by asset reserve (issuer).
+   - *tx2*: Asset clawback transaction from asset reserve to address = `address`, amount = `amount`.
+
+2. [*kill()*](examples/permissioned-token/scripts/admin/kill.js): Application call to controller with a) appArg `str:kill` and b) foreignAsset `assetIndex`- signed by asset manager.
+
+3. [*forceTransfer (fromAddress, toAddress, amount)*](examples/permissioned-token/scripts/admin/force-transfer.js): Group of 4 transactions
+   - *tx1*: Call to controller smart contract with a) application arg `str:force_transfer` b) foreign Asset: `assetIndex` - signed by asset manager.
+   - *tx2*: Asset clawback transaction from `fromAddress` to `toAddress`, amount = `amount`.
+   - *tx3*: Payment transaction to clawback (contract account) with amount >= fee of *tx2*
+   - *tx4*: Call to permissions smart contract with a) application arg `str:transfer` b) app accounts: [`fromAddress`, `toAddress`].
+
+    **NOTE**: *tx3*, *tx4* can be signed by anyone but they should be present in group. If receiver of `forceTransfer` is the current asset reserve OR if *tx4* is asset config txn (to update reserve address), then it can bypass permission checks.
```

Comment:
```suggestion
    **NOTE**: *tx3*, *tx4* can be signed by anyone but they must be present in group (they validate conditions). The signer will pay transaction fees. If receiver of `forceTransfer` is the current asset reserve OR if *tx4* is asset config txn (to update reserve address), then a permissions smart contract call is not required.
```

Reply:
Maybe we should reword it. In the second sentence of the note, a reader may have impression that we are still talking about `forceTransfer` - so why do we talk about asset config tx in this context? I think we should move that part to the use case below.

Reply:
ok, makes sense